lib/elixir/lib/calendar.ex

```diff
@@ -194,6 +234,16 @@ defmodule Date do
   end
 
   @doc """
+  Returns the number of days in a week.
+
+  Always returns 7 days in a week irrespective of the
+  calendar in use.
+  """
+  def days_in_a_week() do
```

Comment:
Should we impose this constraint? Which calendars will we reject by making such choice? Why not ask the calendar for this value?